nautobot/core/events/__init__.py

```diff
@@ -12,6 +16,32 @@
 logger = logging.getLogger(__name__)
 
 
+def load_event_brokers(settings):
+    """Process plugins and log errors if they can't be loaded."""
+    if not getattr(settings, "NAUTOBOT_EVENT_BROKERS", None):
+        return
+
+    for publisher_name, publisher in settings.NAUTOBOT_EVENT_BROKERS.items():
+        options = publisher.get("OPTIONS", {})
+        include_topics = None
+        exclude_topics = None
+        if topics := publisher.get("TOPICS"):
+            include_topics = topics.get("INCLUDE")
+            exclude_topics = topics.get("EXCLUDE")
```

Comment:
```suggestion
        topics = publisher.get("TOPICS", {})
        include_topics = topics.get("INCLUDE", [])
        exclude_topics = topics.get("EXCLUDE", [])
```

Might actually make sense for `include_topics` to default to `["*"]` if unset, actually?